q2_types/distance_matrix/tests/test_format.py

```diff
@@ -0,0 +1,48 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016--, QIIME development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+import os.path
+import shutil
+import unittest
+
+from q2_types.distance_matrix import LSMatFormat, DistanceMatrixDirectoryFormat
+from q2_types.testing import TestPluginBase
+
+
+class TestFormats(TestPluginBase):
+    package = 'q2_types.distance_matrix.tests'
+
+    def test_lsmat_format_validate_positive(self):
+        filenames = ('distance-matrix-1x1.tsv', 'distance-matrix-2x2.tsv',
+                     'distance-matrix-NxN.tsv')
+        for filename in filenames:
+            filepath = self.get_data_path(filename)
+            format = LSMatFormat(filepath, mode='r')
+
+            # Should not error.
+            format.validate()
+
+    def test_lsmat_format_validate_negative(self):
+        filepath = self.get_data_path('not-lsmat')
+        format = LSMatFormat(filepath, mode='r')
+
+        with self.assertRaisesRegex(ValueError, 'LSMat'):
+            format.validate()
+
+    def test_distance_matrix_directory_format(self):
```

Comment:
Add a comment saying that this test is mostly to confirm that a working definition exists